development/tools/cmd/yamlmerge/README.md

```diff
@@ -0,0 +1,30 @@
+# YAML merge
```

Comment:
Why do we need to merge the YAML files? Can't we just run `kubectl apply -f` in the components folder to deploy all the resources to the cluster?

Reply:
We can, k8s has a setup like this and they use bazel to merge it into starter.yaml. The problem would be just, that if we move all the `components` into the main folder, the amount of files in that folder would just grow and it would be hard to keep an overview. If we have the merger tool, we can work on the separate files, merge them into starter.yaml at the end and do `kubectl apply -f` to the main folder (assuming it does not include yaml files from subfolders)